plugins/backstage-plugin-flux/README.md

```diff
@@ -282,9 +282,20 @@ kubernetes:
           caData: LS0tLS1CRUdJTiBDRVJUSUZJQ0...
 ```
 
+You can also update the configuration to restrict the access to the sync/suspend/resume of resources. To do this, simple set readOnly to true.
+
+```yaml
+# app-config.yaml
+
+gitops:
+  readOnly: true
+```
+
+The UI will reflect this by showing the buttons corresponding to sync/suspend/resume as disabled.
```

Comment:
We should probably call out here this is only to disable buttons in the UI which should be used as **part** of a readOnly solution which should also include disabling the k8s-proxy https://backstage.io/docs/features/kubernetes/proxy 

> [!IMPORTANT]
> Enabling `readOnly` mode disables the buttons in the UI and provides feedback to the user that resources cannot be changed. However this is only part of a full `readOnly` solution. The [Backstage Kubernetes Proxy](https://backstage.io/docs/features/kubernetes/proxy) should be configured to deny users from changing resources.

Kind of thing? @bigkevmcd